phoenix-core/src/main/java/org/apache/phoenix/mapreduce/PhoenixRecordReader.java

```diff
@@ -108,6 +109,7 @@ public void initialize(InputSplit split, TaskAttemptContext context) throws IOEx
         final PhoenixInputSplit pSplit = (PhoenixInputSplit)split;
         final List<Scan> scans = pSplit.getScans();
         try {
+            LOG.info("Generating iterators for " + scans.size() + " scans in keyrange: " + pSplit.getKeyRange());
```

Comment:
Shall we remove this log or change it to debug log? The same question for the other two info logs you added below.

Reply:
Record reader is initialized once per mapper job and the map jobs are limited by number of regions. Also these logs will be distributed on several machines and not one. It would be good to have some insight here on the iterators each reader is trying to generate.

Reply:
OK, that's fine then.